apps/web/next.config.mjs

```diff
@@ -9,17 +9,10 @@ const baseNextConfig = {
 		TELEGRAM_BOT_TOKEN: process.env.TELEGRAM_BOT_TOKEN,
 	},
 	eslint: {
-		ignoreDuringBuilds: true
+		ignoreDuringBuilds: true,
 	},
 };
 
-const withPWA = pwa({
```

Comment:
@leerob , may i know why PWA configuration has been removed?

Reply:
- next-pwa uses Workbox
- Workbox is no longer maintained
- next-pwa requires custom webpack configuration, making it incompatible with turbopack

I wanted to test on this PR if Turbopack works for you all, and what the performance improvements are. You might consider exploring alternative options based on what you were hoping to get from a PWA (was it install to your home screen? E.g. this is now built-into iOS).

Reply:
next-pwa was integrated with thought of adding push notifications in near future

Reply:
what's a good alternative to next-pwa?

Reply:
This will help https://serwist.pages.dev/docs/next